<launch>
  <node name="rgb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="rbg_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="rgb_image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/rgb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>

  <!-- event camera driver -->
  <node name="dvxplorer_ros_driver" pkg="dvxplorer_ros_driver" type="dvxplorer_ros_driver" />
  <!-- events visualization -->
  <node name="dvs_renderer" pkg="dvs_renderer" type="dvs_renderer">
    <!-- <param name="display_method" value="grayscale"/> -->
    <param name="display_method" value="red-blue"/>
    <remap from="events" to="/dvs/events" />
    <remap from="camera_info" to="/dvs/camera_info" />
  </node>
  <node name="event_image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/dvs_rendering"/>
    <param name="autosize" value="true" />
  </node>
</launch>
